src/plugins/walletConnectToDapps/utils.ts

```diff
@@ -57,13 +57,15 @@ export const getGasData = (
   fees: FeeDataEstimate<EvmChainId>,
 ) => {
   const { speed, customFee } = customTransactionData
-  return speed === 'custom' && customFee?.baseFee && customFee?.baseFee
+  return speed === 'custom' &&
+    bnOrZero(customFee?.baseFee).gt(0) &&
+    bnOrZero(customFee?.priorityFee).gt(0)
```

Comment:
Defined isn't enough, `baseFee` and `priorityFee` may be `'0'` as tested with Uniswap